infrastructures/docker/src/main/java/org/eclipse/che/workspace/infrastructure/docker/model/DockerContainerConfig.java

```diff
@@ -135,6 +139,7 @@ public DockerContainerConfig withBuild(DockerBuildContext build) {
     /**
      * Override the default entrypoint.
      */
+    @Nullable
```

Comment:
These specific fields should be null by default because non-null values can break start of container